### ISSUE TITLE
Unconscious Command+Shift+R reloads Hyper breaking all ongoing processes

### ISSUE THREAD

**Issue by santisiri** [zeit/hyper#2987]
- [X] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [X] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **Mac OS X El Capitan**: 
- **Hyper.app 2.0.0**: 
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: 
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: 

## Issue

`window:reloadFull": "command+shift+r"` i find myself constantly doing this to reload the browser until unconsciously do it on Hyper and kills all my processes and must restart again. 

A popup warning "Do you want to do a full reload" would be good to prevent this.

**Comment by albinekb** [comment]
Maybe better suited as a plugin? 🤔 I would not want a warning every-time to reload, it's really useful when debugging plugins/themes 🤔 Maybe if we could only show a dialog if there's a process running in any of the tabs?

**Comment by dwehrmann** [comment]
Yeah, same here. Processes keep running and need to be killed, whereas the info in the terminal is gone. This needs improvement. A warning would be sufficient I think. There's still CTRL-R that can be kept without warning as it keeps things running.
